### ISSUE TITLE
Multiple Assignment

### ISSUE THREAD

**Issue by Max-ChenFei** [mitmproxy/pdoc#388]
https://github.com/mitmproxy/pdoc/blob/a57cdb235dff43ceb8d512209b8aebf252175b3f/pdoc/doc_ast.py#L108-L109
Here only support a single assignment for the variable.

Will we support multiple assignment?

**Comment by mhils** [comment]
The code you linked is used to extract variable docstrings from the abstract syntax tree. Here's an example:
```python
default_timeout = 42
"""
The default timeout for HTTP requests, in seconds.
"""
```

I'm not sure if supporting multiple assignments make sense here? Is the docstring then referring to the first variable, the second variable, ...? Do you have a real-world example that motivates your question?

**Comment by mhils** [comment]
Right - I think for pdoc variables without docstrings are a bit out of scope. For your specific case I'd recommend forking the `doc_ast` parts and adding your own patches there, as you already seem to be doing. :)

**Comment by Max-ChenFei** [comment]
Thanks a lot. I will folk the pdoc and add my own patches. Should I create PR?

**Comment by mhils** [comment]
I don't think we need this functionality in pdoc and it creates a maintenance burden, so I'd kindly ask you to just maintain your fork. No need to send a PR. :)

**Comment by Max-ChenFei** [comment]
Ok, no problem.
